src/context/auth.tsx

```diff
@@ -57,6 +59,7 @@ export const AuthProvider: FC<ProviderProps> = ({ children }) => {
     <AuthContext.Provider
       value={{
         isAuthenticated: (settings && settings.auth !== "untrusted") ?? false,
+        isOidc: settings?.auth_user_method === "oidc",
```

Comment:
I see, so I assume if a valid certificate exists but the user authenticated using oidc, then oidc will be the preferred auth method? I guess maybe it's not typical that an lxd server would be set up with both tls and oidc authentication as it might relate to company policies etc

Reply:
just tested the case by adding back my certificate while I'm logged in with oidc. When logging out, I get instantly logged back in haha, I think that's the expected behaviour though since both auth methods are valid

Reply:
It is possible both auth methods are available, I think it is even not possible to disable tls.

If LXD server prefers oidc or tls, when presented with both at the same time -- from testing it seems OIDC is preferred: But then again this is a very unlikely scenario. In the UI we wouldn't show a login to a tls user. You could fetch the `/oidc/login` route manually and be authenticated with both. The UI handles this well, and it seems the backend announces OIDC as auth in this case. The UI will show a logout option in the nav, when used the OIDC auth is removed, but the user stays login as the backend falls back to tls. All in all I think that is expected, though highly unusual to use both in parallel.